pyproject.toml

```
@@ -27,6 +27,9 @@ classifiers = [
 requires-python = ">= 3.11, < 4"
 dependencies = [
   "typing-extensions >= 4.12.2, < 5",
+  "frequenz-api-assets @ git+https://github.com/frequenz-floss/frequenz-api-assets.git@v0.0.2",
```

```suggestion
  "frequenz-api-assets @ git+https://github.com/frequenz-floss/frequenz-api-assets.git@28a4acf547cc84aeed151f882f8c1442af94a8f7",
```
Version 0.0.2 does not include the latest changes to the common API; however, this hash version does. I recommend using it instead.

---

Ah great, thank you!